sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Fixed
 
 - Fixed concurrency issue in our challenge-based authentication policy ([#9737](https://github.com/Azure/azure-sdk-for-net/issues/9737))
+- Fixed The return value of issuer name always be null ([#10908](https://github.com/Azure/azure-sdk-for-net/issues/10908))
```

Comment:
That sentence doesn't make sense. Please reword with what/how you fixed rather than "Fixed <bug title>". E.g. "Fixed an issue where the issuer name was always null".